### ISSUE TITLE
Change factuur versturen button to only generate a PDF and let the user download it

### ISSUE THREAD

**Issue by Ellen-Wittingen** [csvalpha/sofia#843]
Currently the `factuur versturen` knop in the invoice list in [http://localhost:5000/invoices](http://localhost:5000/invoices) generates a pdf and then emails it to the user that the invoice is for. 

We do not like this as then you do not notice an email is sent or you cannot alter it. 

Instead, the button can be renamed to `Factuur downloaden` which would generate the pdf and download it to the computer of the person that pressed the button

**Comment by Ellen-Wittingen** [comment]
Requires #842 to be fixed first

**Comment by wilco375** [comment]
I don't think this is desired because the email also includes  a link to directly pay the invoice via iDEAL and it saves the treasurer a lot of time from having to manually compose emails. We could add an extra button to just download the PDF though.
